examples/cms-builder-io/next.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  images: {
+    domains: ['cdn.builder.io'],
+  },
+  async headers() {
+    return [
+      {
+        source: '/:path*',
+        headers: [
+          {
+            key: 'Content-Security-Policy',
+            value:
+              'frame-ancestors https://*.builder.io https://builder.io http://localhost:1234',
+          },
+        ],
+      },
+    ]
+  },
+  env: {
+    // expose env to the browser
+    BUILDER_PUBLIC_KEY: process.env.BUILDER_PUBLIC_KEY,
```

Comment:
Oneeee more small thing then we can ship this! Apologies I didn't catch it.

You can remove this in favor of doing `NEXT_PUBLIC_` as a prefix for the env var. This automatically will expose it on the client side. So `NEXT_PUBLIC_BUILDER_KEY` for example.